[2024-09-22T11:28:29.452+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-22T11:28:29.509+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_dynamic_task_mapping_with_no_taskflow_operators.add_one scheduled__2024-09-20T00:00:00+00:00 map_index=0 [queued]>
[2024-09-22T11:28:29.525+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_dynamic_task_mapping_with_no_taskflow_operators.add_one scheduled__2024-09-20T00:00:00+00:00 map_index=0 [queued]>
[2024-09-22T11:28:29.526+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-22T11:28:29.745+0000] {taskinstance.py:2330} INFO - Executing <Mapped(AddOneOperator): add_one> on 2024-09-20 00:00:00+00:00
[2024-09-22T11:28:29.789+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=932) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-22T11:28:29.800+0000] {standard_task_runner.py:64} INFO - Started process 939 to run task
[2024-09-22T11:28:29.798+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping_with_no_taskflow_operators', 'add_one', 'scheduled__2024-09-20T00:00:00+00:00', '--job-id', '2615', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_dynamic_task_mapping_with_no_taskflow_operators.py', '--cfg-path', '/tmp/tmpjnvk2fz4', '--map-index', '0']
[2024-09-22T11:28:29.826+0000] {standard_task_runner.py:91} INFO - Job 2615: Subtask add_one
[2024-09-22T11:28:30.298+0000] {task_command.py:426} INFO - Running <TaskInstance: example_dynamic_task_mapping_with_no_taskflow_operators.add_one scheduled__2024-09-20T00:00:00+00:00 map_index=0 [running]> on host 1d2bd9c1668e
[2024-09-22T11:28:31.205+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_dynamic_task_mapping_with_no_taskflow_operators' AIRFLOW_CTX_TASK_ID='add_one' AIRFLOW_CTX_EXECUTION_DATE='2024-09-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-20T00:00:00+00:00'
[2024-09-22T11:28:31.207+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-22T11:28:31.209+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-22T11:28:31.817+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_dynamic_task_mapping_with_no_taskflow_operators, task_id=add_one, run_id=scheduled__2024-09-20T00:00:00+00:00, map_index=0, execution_date=20240920T000000, start_date=20240922T112829, end_date=20240922T112831
[2024-09-22T11:28:32.562+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-09-22T11:28:32.676+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-22T11:28:33.128+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
